department-of-veterans-affairs/abd-vro
# SecRel Code Scan Fix: update transient dependencies for hapi-fhir-server-openapi

## What was the problem?
Some code scanning alerts triggered for some transient dependencies of the hapi-fhir-server-openapi reference (see below slack thread for finer details):

- [alert 1](https://github.com/department-of-veterans-affairs/abd-vro-internal/security/code-scanning/638)
- [alert 2](https://github.com/department-of-veterans-affairs/abd-vro-internal/security/code-scanning/638)
- [alert 3](https://github.com/department-of-veterans-affairs/abd-vro-internal/security/code-scanning/636)
- [alert 4](https://github.com/department-of-veterans-affairs/abd-vro-internal/security/code-scanning/635)

Associated tickets or Slack threads:
- [slack thread](https://dsva.slack.com/archives/C031RRFCHV1/p1674589664138679)

## How does this fix it?[^1]
This PR updates those dependencies, and fixes the problem. 

Successful scan: https://github.com/department-of-veterans-affairs/abd-vro-internal/actions/runs/4001127339

## How to test this PR
- Run a successful secrel scan


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
